src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -84,6 +84,9 @@ private void DeterminePesterVersion()
             PesterSymbolReference pesterSymbol,
             ScriptFile scriptFile)
         {
+            // A value of null is a signal to PSES that the available Pester version does not support
+            // running Describe blocks by name (the test name will used instead then)
+            int? describeBlockLineNumber = _pesterV4_6_0_OrHigherAvailable ? pesterSymbol.ScriptRegion.StartLineNumber : default(int?);
```

Comment:
Isn't `default(int?)` just `null`?  I get using `default(T)` in a generic class or method but not sure it is needed here.  Unless I'm missing something - which is ENTIRELY possible.  :-)

Reply:
Yes, but...
![image](https://user-images.githubusercontent.com/9250262/51573035-8ca88b00-1e9f-11e9-9041-35c50a583db0.png)

Reply:
Yes, `default(int?)` is null. I changed it to the following, which is probably the most readable version by casting the first `int` to `int?`:
```csharp
int? describeBlockLineNumber = _pesterV4_6_0_OrHigherAvailable ? (int?)pesterSymbol.ScriptRegion.StartLineNumber : null;
```